File: src/com/yentar/structuralPatterns/classAdapter/EmployeeDataParser.java
package com.yentar.structuralPatterns.classAdapter;

public class EmployeeDataParser implements JsonParser {

    @Override
    public String parse() {
        String result = "{" + "\n";
        result += "\t name: Mehmet" + "\n";
        result += "\t age: 99" + "\n";
        result += "}";
        return result;
    }
}

File: src/com/yentar/structuralPatterns/classAdapter/EmployeeDataParser.kt
package com.yentar.structuralPatterns.classAdapter

class EmployeeDataParser : JsonParser {
    override fun parse(): String {
        var result = """
            {
            
            """.trimIndent()
        result += """	 name: Mehmet
"""
        result += """	 age: 99
"""
        result += "}"
        return result
    }
}